### ISSUE TITLE
Is this library still maintained?

### ISSUE THREAD

**Issue by karesztrk** [evoluhq/evolu#501]
Hey, 
Is this library still maintained and under active development? According to the docs, it will be rewritten (without Effect), but the latest commits are either CI or Documentation updates. 
Do we have a roadmap?

**Comment by steida** [comment]
Hi, yes, it is!

Several months ago, I realized that the current approach (MerkleTree) does not scale well and is probably impossible to fix. I researched a better algorithm and implemented it with many tests. I also created the Evolu Binary Protocol and fixed a few other issues. Recently, for reasons mentioned elsewhere, I decided to remove Effect and Effect Schema. I also created Evolu Type and a few helpers, entirely replacing the need for Effect. That change, by the way, opened further DX improvements and paved the way for the upcoming Evolu Collaboration. I updated Eslint to 9; currently, I'm reimplementing evolu RN with op-sqlite support. We also have a new website. I collaborate on these changes with selected Evolu users in private discord inner-circle group.

I want to highlight that the new Evolu is backward compatible. We have auto-migration; only internal tables have been changed. App tables are untouched.

The current Evolu works well; only deps could be updated, but the release has a higher priority.
